lib/v2/reuters/router.js

```diff
@@ -0,0 +1,4 @@
+module.exports = (router) => {
+    router.get('/investigates', require('./investigates'));
```

Comment:
```suggestion
    router.get('/investigates', require('./investigates'));
    router.get('/reuters/channel/:site/:channel', require('./migration_prompt')); // deprecated
    router.get('/reuters/theWire', require('./migration_prompt')); // deprecated
```
Then raise an error to prompt users to read the doc.